gef-ui/server/utils/helpers.js

```diff
@@ -249,13 +259,15 @@ const summaryItemsConditions = (summaryItemsObj) => {
   const value = typeof details[item.id] === 'number' || typeof details[item.id] === 'boolean' ? details[item.id].toString() : details[item.id];
   const isCoverStartOnSubmission = id === 'coverStartDate' && shouldCoverStartOnSubmission;
   // column keys to display change if facility has been changed to issued
-  const changedToIssueShow = id === 'name' || id === 'coverStartDate' || id === 'coverEndDate' || id === 'issueDate';
+  const changedToIssueShow = id === 'name' || id === 'coverStartDate' || id === 'coverEndDate' || id === 'issueDate' || id === 'hasBeenIssued';
```

Comment:
Compound condition complexity is over 3, should be a helper function in my opinion.

Reply:
fixed